trio/_tests/test_threads.py

```diff
@@ -881,15 +884,15 @@ async def test_trio_token_weak_referenceable() -> None:
     assert token is weak_reference()
 
 
-async def test_unsafe_cancellable_kwarg() -> None:
+async def test_unsafe_abandon_on_cancel_kwarg() -> None:
     # This is a stand in for a numpy ndarray or other objects
     # that (maybe surprisingly) lack a notion of truthiness
     class BadBool:
         def __bool__(self) -> bool:
             raise NotImplementedError
 
     with pytest.raises(NotImplementedError):
-        await to_thread_run_sync(int, cancellable=BadBool())  # type: ignore[arg-type]
+        await to_thread_run_sync(int, abandon_on_cancel=BadBool())  # type: ignore[arg-type]
```

Comment:
Because `to_thread_run_sync` is an overloaded function, the error code associated with it changed from `arg-type` to `call-overload`.

This also applies to line 586.